frontend/src/pages/settings/users/_userId/index.vue

```diff
@@ -0,0 +1,532 @@
+<template>
+  <SettingsPage page-title="settings.users.users">
+    <template #actions>
+      <VBtn
+        variant="elevated"
+        href="https://jellyfin.org/docs/general/server/devices.html"
```

Comment:
This needs to be updated!
```suggestion
        href="https://jellyfin.org/docs/general/server/users/"
```